core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -92,12 +93,21 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission1),
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission2))
 
-    Get(s"/admin/submissions") ~>
-      sealRoute(services.adminRoutes) ~>
-      check {
-        assertResult(StatusCodes.OK) { status }
-        assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
+    withStatsD {
+      Get("/admin/submissions") ~>
+        sealRoute(instrumentRequest { services.adminRoutes }) ~>
+        check {
+          assertResult(StatusCodes.OK) {
+            status
+          }
+          assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
+        }
+    } { capturedMetrics =>
+      val expected = expectedHttpRequestMetrics("get", "admin.submissions", StatusCodes.OK.intValue, 1)
+      assert {
```

Comment:
In general I like using Scalatest matchers instead of `assert(boolean)` because it gives nicer error messages. Unfortunately it looks like our current version of scalatest doesn't have a great way of verifying subsets (it was added in 3.0, see https://groups.google.com/forum/#!topic/scalatest-users/SyyW8RC1pfA).

That said, I would probably still prefer something like this:
```
val capturedSet = capturedMetrics.toSet
expectedHttpRequestMetrics("get", "admin.submissions", StatusCodes.OK.intValue, 1).foreach { m =>
  capturedSet should contain (m)
}
```
Then at least you would see _which_ expected metric was not contained in the captured set in the error message.